### PR TITLE
#1314 Version number overlaping sidebar

### LINKED ISSUE #1314: Version Number Overlapping Icons on Sidebar
### Browser(s)

Chrome, Firefox, Safari

### Platform(s)

Desktop, iOS, Android

### Version(s)

develop

### Observed Behavior

Version # is overlapping icons when the screen is too small
image

### Expected Behavior

It should just be below the last icon

### Steps to Reproduce

1. Open Finishline on Develop
2. Make your screen a lot smaller
3. 

### Screenshots (as needed)

_No response_

### PR DESCRIPTION
**Pull request by willturner106**
## Changes

I fixed the issue where the version number overlaps the sidebar when the window gets to small. I did this by changing the position of the version number from "absolute" to "relative" in the css file. I then moved it outside of the sidebar box so it stays visible as the window gets smaller. I also added some padding on top of the version number to keep the spacing.

## Test Cases

1) Screen is really small vertically
 - The side links turn into a scrollbar, so that the version number doesn't overlap the links.

## Screenshots

Focusing on the appearance of the navigation bar and version number on the left

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1314

### CODE REVIEW

**Review by tigerhippo — COMMENTED** [review]
This looks great! I'm gonna check in with Reid about the UI design but otherwise I'd say it's pretty much ready to merge. Nice work!

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -75,8 +75,8 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
         {linkItems.map((linkItem) => (
```
Comment (RChandler234) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:75]: ok so you should add these props to the Box in line 74 (here's a guide to flexbox if u wanna look at it https://css-tricks.com/snippets/css/a-guide-to-flexbox/):
```
        display="flex"
        flexDirection={'column'}
        flex={1}
        justifyContent={'space-between'}
```

Comment (RChandler234) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:75]: also make sure to wrap these linkItems in a <Box> </Box>

File: src/frontend/src/stylesheets/layouts/sidebar/sidebar.module.css [hunk 2]
```diff
@@ -14,10 +14,11 @@
...
 
 .versionNumber {
-  position: absolute;
+  position: relative;
   text-align: center;
   color: #ffffff;
-  width: 100%;
   bottom: 0;
+  width: 100%;
   height: 6em;
```
Comment (RChandler234) [inline src/frontend/src/stylesheets/layouts/sidebar/sidebar.module.css:22]: delete lines 20, 22, and 23, and make sure to add in a marginBottom (like 20-30ish) so the version number isn't right on the bottom of the screen

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (willturner106) [reply src/frontend/src/layouts/Sidebar/Sidebar.tsx:75]: Sorry, wrap the linkitems in a what?

Comment (RChandler234) [reply src/frontend/src/layouts/Sidebar/Sidebar.tsx:82]: `<Box>` sorry github it weird ab putting code in comments

File: src/frontend/src/stylesheets/layouts/sidebar/sidebar.module.css [hunk 3]
```diff
@@ -14,10 +14,9 @@
...
 
 .versionNumber {
-  position: absolute;
+  position: relative;
   text-align: center;
   color: #ffffff;
   width: 100%;
-  bottom: 0;
-  height: 6em;
+  padding-bottom: 3em;
```
Comment (RChandler234) [inline src/frontend/src/stylesheets/layouts/sidebar/sidebar.module.css:21]: padding on the bottom should be a lil less

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -71,12 +71,19 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
...
+        display="flex"
+        flexDirection={'column'}
+        flex={1}
+        justifyContent={'space-between'}
+      >
         {linkItems.map((linkItem) => (
           <NavPageLink {...linkItem} open={open} />
         ))}
-        <Typography className={styles.versionNumber}>4.0.0</Typography>
       </Box>
```
Comment (RChandler234) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:85]: the Typography should be inside this

File: src/frontend/src/layouts/Sidebar/Sidebar.tsx [hunk 5]
```diff
@@ -71,12 +71,19 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
...
-      <Box overflow={'auto'} sx={{ overflowX: 'hidden' }}>
+      <Box
+        overflow={'auto'}
+        sx={{ overflowX: 'hidden' }}
+        display="flex"
+        flexDirection={'column'}
+        flex={1}
+        justifyContent={'space-between'}
+      >
         {linkItems.map((linkItem) => (
```
Comment (RChandler234) [inline src/frontend/src/layouts/Sidebar/Sidebar.tsx:82]: all the linkItems should be under one box, so one box for all of the linkitems together

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥 
Nice Job

**Pull request merged**